tests/from.bats

```diff
@@ -287,3 +287,11 @@ load helpers
   cid=$(buildah from --name=${container_name} --pull --signature-policy ${TESTSDIR}/policy.json alpine)
   buildah --debug=false inspect --format '{{.Container}}' ${container_name}
 }
+
+@test "from cidfile test" {
+  buildah from --cidfile output.cid --pull --signature-policy ${TESTSDIR}/policy.json alpine
+  cid=$(cat output.cid)
+  run buildah --debug=false containers -f id=${cid}
+  [ "$status" -eq 0 ]
+  buildah rm ${cid}
```

Comment:
A touch up for later would be to add a status check for line 296 too.  That way you'll know that you go the cid appropriately and it removed the container rather than silently failing here.